### PR TITLE
Turn off hash merge, add gh_user_no_sudo

### PR DESCRIPTION
**Pull request by sefk**
add users without sudo (gh_user_no_sudo)

It's handy to be able to create users that _don't_ have sudo rights. Here at Stanford we use this to add users to our VPC bastion box (ssh proxy) so these users can use this machine for ssh tunneling, but I don't want to give those users the keys to the kingdom.

This let's me configure a playbook like this.

```
   roles:
     - common
     - supervisor
     - role: gh_users
       gh_users:
         - su1
         - su2
         - su3
       gh_users_no_sudo:
         - normal1
         - normal2
```

The new gh_users_no_sudo list can be empty.

@jbau or @jrbl.

### CODE REVIEW

File: playbooks/roles/gh_users/tasks/main.yml [hunk 1]
```diff
@@ -27,22 +29,28 @@
...
   user:
     name={{ item }} groups=gh
     shell=/bin/bash
   with_items: gh_users
 
+- name: gh_users | create non-sudo github users
+  user:
+    name={{ item }}
+    shell=/bin/bash
+  with_items: gh_users_no_sudo
```
Comment (sefk) [inline playbooks/roles/gh_users/tasks/main.yml:42]: note just the no_sudo variant here, these users don't have a group=gh

**Comment by sefk** [comment]
Here's the log excerpt when run for us:

TASK: [gh_users | gh_users | create sudo github users] ***********************\* 
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=sefk)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=jbau)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=jrbl)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=ali123)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=caesar2164)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=dcadams)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=nparlante)

TASK: [gh_users | gh_users | create non-sudo github users] *******************\* 
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=jinpa)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=gbruhns)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=paepcke)
changed: [ip-10-0-0-17.us-west-1.compute.internal] => (item=akshayak)

TASK: [gh_users | gh_users | create .ssh directory] **************************\* 
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=sefk)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=jbau)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=jrbl)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=ali123)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=caesar2164)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=dcadams)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=nparlante)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=jinpa)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=gbruhns)
ok: [ip-10-0-0-17.us-west-1.compute.internal] => (item=paepcke)
changed: [ip-10-0-0-17.us-west-1.compute.internal] => (item=akshayak)

File: playbooks/roles/gh_users/tasks/main.yml [hunk 2]
```diff
@@ -10,6 +10,8 @@
 #     gh_users:
 #      - joe
 #      - mark
+#     gh_users_no_sudo:
+#      - dave_the_janitor
```
Comment (jbau) [inline playbooks/roles/gh_users/tasks/main.yml]: Dave is a valuable team member.  Also, the fancy automated floor mop requires sudo.

**Comment by jrbl** [comment]
lgtm

File: playbooks/roles/gh_users/tasks/main.yml (hunk 2 above)
Comment (sefk) [inline playbooks/roles/gh_users/tasks/main.yml]: Indeed.  made a better example, now he's just `tourist_dave`

**Comment by sefk** [comment]
Thanks for reviewing, merging and will do a separate PR into master.

**Pull request merged**